engine/flink/executor/src/main/scala/pl/touk/nussknacker/engine/process/compiler/FlinkProcessCompilerData.scala

```diff
@@ -38,9 +38,17 @@ class FlinkProcessCompilerData(
 
   def open(runtimeContext: RuntimeContext, nodesToUse: List[_ <: NodeData]): Unit = {
     val lifecycle = compilerData.lifecycle(nodesToUse)
-    lifecycle.foreach {
-      _.open(FlinkEngineRuntimeContextImpl(jobData, runtimeContext))
+    componentUseCase match {
+      case ComponentUseCase.TestRuntime =>
+        lifecycle.foreach {
```

Comment:
DRY. Only engine runtime context creation looks different for both cases. Opening behave the same